components/core/src/clp/EncodedVariableInterpreter.cpp

```
@@ -196,6 +196,23 @@ void EncodedVariableInterpreter::convert_encoded_float_to_string(
     value[value_length - 1 - decimal_pos] = '.';
 }
 
+auto EncodedVariableInterpreter::wildcard_string_could_be_representable_integer_var(
+        std::string_view value
+) -> bool {
+    return false == std::ranges::any_of(value, [](char const c) -> bool {
+               return false == (('0' <= c && c <= '9') || c == '-' || c == '?' || c == '*');
+           });
+}
+
+auto EncodedVariableInterpreter::wildcard_string_could_be_representable_float_var(
```

Cppcheck (2.19.0)

[style] The function 'wildcard_string_could_be_representable_float_var' is never used.

(unusedFunction)